sdk/digitaltwins/Azure.DigitalTwins.Core/src/DigitalTwinsClient.cs

```diff
@@ -506,7 +506,7 @@ public virtual async Task<Response> UpdateDigitalTwinAsync(string digitalTwinId,
         /// <param name="jsonPatchDocument">The application/json-patch+json operations to be performed on the specified digital twin.</param>
         /// <param name="ifMatch">Optional. Only perform the operation if the entity's ETag matches this optional ETag or * (<see cref="ETag.All"/>) is provided.</param>
         /// <param name="cancellationToken">The cancellationToken.</param>
-        /// <returns>The http response <see cref="Response{T}"/>.</returns>
+        /// <returns>The HTTP response <see cref="Response{T}"/>. This response object includes an HTTP header that gives you the new ETag for this resource.</returns>
```

Comment:
I like the fact that we are specifying the Etag information, but that's not all, there are a lot of other information in this object. is there a reason we are calling out ETag ?

Reply:
We had a customer issue (linked in the description of this PR) where they were asking how to get this ETag after updating a resource without having to get the resource again. Since this is a particularly important header to pay attention to, I wanted to call it out.